index.html

```diff
@@ -298,10 +298,12 @@ <h3>Requirements for advancing the maturity level of features</h3>
     its implementability has been demonstrated.</p>
     <p>When considering suitability to advance any feature beyond
     Candidate Recommendation, at least two independent factors of verification
-    MUST be demonstrated, which may come from any of:</p>
+    for each normative requirement
+    MUST be demonstrated, which may come, as relevant for that requirement,
+    from any of:</p>
     <ul>
       <li>Presentation implementation</li>
-      <li>Content</li>
+      <li>Content producing implementation</li>
       <li>Validating implementation</li>
     </ul>
     <p>For example, a feature MAY be advanced beyond Candidate Recommendation
```

Comment:
```suggestion

```

An additional recommendation from the council meeting was to remove this example entirely. At a minimum, the `MAY` should be changed to "may". (The github UI is not letting me do a suggested change across multiple lines for the lines that were not originally included in PR diff for some reason.)

Reply:
I've changed MAY to may in 7860cb0. Please note that this example was subject to a FO from Google that we resolved by consensus with the current wording.

Reply:
Assuming silence is assent, and resolving this conversation.

Reply:
I think this example may work at cross-purposes to the other changes here, so I think it would be better to remove it.